File: Java/modul6_kel46_settergetter.java
public class modul6_kel46_settergetter { //membuat class
    //deklarasi atribut
    private String merk;
    private String tipe;
    private String warna;
    private String jenisBBM;

    //membuat setter
    public void setMerk(String merk) {
        this.merk = merk;
    }
    public void setTipe(String tipe) {
        this.tipe = tipe;
    }
    public void setWarna(String warna) {
        this.warna = warna;
    }
    public void setJenisBBM(String jenisBBM) {
        this.jenisBBM = jenisBBM;
    }
    
    //membuat getter
    public String getMerk(){
        return merk;
    }
    public String getTipe(){
        return tipe;
    }
    public String getWarna(){
        return warna;
    }
    public String getJenisBBM() {
        return jenisBBM;
    }
}

File: Php/modul6_kel46_settergetter.php
<?php
Class Mobil { //membuat kelas
	private $merk;
	private $tipe;
	private $warna;
	private $jenisBBM;
	function setMerk($merk) { //membuat setter
		$this->merk = $merk;
	}
	function setTipe($tipe){
		$this->tipe = $tipe;
	}
	function setWarna($warna){
		$this->warna = $warna;
	}
	function setJenisBBM($jenisBBM){
		$this->jenisBBM = $jenisBBM;
	}
	function getMerk() { //membuat getter
		return $this->merk;
	}
	function getTipe() {
		return $this->tipe;
	}
	function getWarna() {
		return $this->warna;
	}
	function getJenisBBM() {
		return $this->jenisBBM;
	}
} // akhir kelas
$mobil_baru = new Mobil(); //membuat objek
//mengatur nilai
$mobil_baru->setMerk("Mitsubishi");
$mobil_baru->setTipe("Mirage");
$mobil_baru->setWarna("merah");
$mobil_baru->setJenisBBM("Pertamax");
//menampilkan data
echo "Saya punya mobil dengan merk ".$mobil_baru->getMerk()." tipe ".$mobil_baru->getTipe();
echo " berwarna ".$mobil_baru->getWarna().".<br> Mobil saya menggunakan bensin berjenis ".$mobil_baru->getJenisBBM();	
?>